pkg/models/sidecar.go

```diff
@@ -7,4 +7,5 @@ type Sidecar struct {
 	Vm        VmType
```

Comment:
Why don't we also add a version for future compatibility matching

Reply:
OK, current strategy I adopted is to write the Version only on sidecar creation, with a different version than the app version (otherwise it's not clear when there needs to be a compatibility check).

Suggesting to apply sidecar version "1.0.0"

Reply:
yep, agreed it's a separate version